### ISSUE TITLE
Fastboot throws error for missing module `velocity`

### ISSUE THREAD

**Issue by trabus** [ember-animation/liquid-fire#513]
Seeing an error in Fastboot as of v0.26.2:
```There was an error running your app in fastboot. More info about the error:
 Error: Could not find module `velocity` imported from `liquid-fire/growable````

It appears to be related to #507 now importing Velocity as it's own module. Previously when using `Ember.$.Velocity` it seems that it was silently failing, but not causing any errors.

I've confirmed locally that always importing the shim and using a noop function seems to solve the issue, though I'm not sure that it's the best solution. I put the following in the shim [here](https://github.com/ember-animation/liquid-fire/blob/master/vendor/shims/velocity.js#L5):
```var Velocity = self.Velocity || self.Ember.$ && self.Ember.$.Velocity || function(){};```

**Comment by ef4** [comment]
Thanks, I agree that making the shim a noop is a good solution. We don't actually try to invoke it in fastboot.

**Comment by trabus** [comment]
@ef4 I'm still tinkering with the noop solution to see exactly what is happening (I just wanted to understand it fully). I'll try and get a PR for you tonight.

Unrelated, but I also noticed some interesting behavior with `liquid-outlet` in fastboot that I'm currently. It is rendering the loading route content in addition to the actual route content (below the actual route content). I'll file a bug if I can figure out what's actually happening.

**Comment by ef4** [comment]
That sounds quite possible. Glimmer2 has different code paths for initial render vs update, and if your app renders the loading state but then quickly updates to the final state, you will go down the update path in fastboot, which has been the subject of some bugs and discussions.

**Comment by ef4** [comment]
Released in 0.26.3, thanks for fixing this!
